Dockerfile

```diff
@@ -0,0 +1,18 @@
+#Using the official ubuntu OS as a Base Image
+FROM ubuntu:16.04
+
+USER root
+
```

Comment:
There are additional libraries that are probably necessary for the system. Check out https://github.com/edx/edx-e2e-tests/blob/master/Vagrantfile